### ISSUE TITLE
What is the difference between gill_propeller and propeller_anemometer?

### ISSUE THREAD

**Issue by kersting** [IEA-Task-43/digital_wra_data_standard#175]
What is the difference between `gill_propeller` and `propeller_anemometer`? Aren't them redundant?

**Comment by heikowestermann** [comment]
`gill_propeller` is a `propeller_anemometer` manufactured by R. M. Young (I never understood why they are not manufactured by Gill), so I think you are right @kersting.

**Comment by kersting** [comment]
@heikowestermann thanks for the feedback!

**Comment by stephenholleran** [comment]
@kersting a `gill_propeller` measures vertical wind speed whereas you can have a horizontal wind speed measured by a `propeller_anemometer`.

**Comment by heikowestermann** [comment]
my definition: `gill_propeller` is a specific device used also as a `vertical_anemometer`. I think there are not many more, except for 3D sonics which measure the vertical component...

**Comment by stephenholleran** [comment]
Hi @abohara @kersting @heikowestermann,

My take on it is that a `gill_propeller` is a device that can measure vertical wind speeds, but it can also measure horizontal wind speeds too. I found a paper from 1975 by Gerald C. Gill himself [DEVELOPMENT AND USE OF THE GILL UVW ANEMOMETER](https://deepblue.lib.umich.edu/bitstream/handle/2027.42/42511/10546_2005_Article_BF02153566.pdf?sequence=1) where he uses the device to measure the 3 components of the wind. 

![image](https://user-images.githubusercontent.com/25622575/173046008-21448fe3-f1da-4306-91c6-930564d177e0.png)

I think a `vertical_anemometer` is still relevant as there are other devices that can measure a vertical wind speed and this can cover all of those. A `gill_propeller` is a commonly known anemometer but it can measure the UVW components of the wind. Therefore, I think both are valid, useful and different.

The device technically should be called a `helicoid_anemometer` according to another paper by Gill [The helicoid anemometer: A long neglected but valuable anemometer](https://www.tandfonline.com/doi/pdf/10.1080/00046973.1973.9648356). So, this Gill guy must have been banging on so long about these helicoid anemometers that they must have just become known as Gill propeller anemometers. I don't think anyone would know what a `helicoid_anemometer` was and so am happy to leave it as `gill_propeller`.

Regarding @abohara suggestion "_if the analyst should know whether a device is a `propeller` or `cup` anemometer. Currently, it seems that by convention, an `anemometer` is cup and `vertical_anemometer` is a prop._" You are right that mostly these are what are used but as @heikowestermann mentioned there are other devices that can measure vertical wind speeds. We have sort of done this with a `propeller_anemometer`, an anemometer that uses a propeller to measure the horizontal wind speed. Including a `vertical_prop_anemometer` would be in the same vane but I don't think we could remove `vertical_anemometer`. Therefore, I'd be inclined to leave it as it is.

**To recap;**
There is a difference between a `gill_propeller` and a `vertical_anemometer`.
I think there is no need to add a `vertical_prop_anemometer`.

Are people happy with this? Can I close the issue?

**Comment by kersting** [comment]
@stephenholleran thanks for all this research! I learned a lot about this topic! I'm happy to close it.
